### PR TITLE
Add initial a71 support

### PR DESCRIPTION
**Pull request by ferZ25**
### Description

initial support for a71, now able to boot simpleinit

### Checklist

* [x] Have you tested the change you are submitting?
* [x] Is the commits history nice and clean?

### CODE REVIEW

File: Platform/Samsung/sm7150/a71.dsc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1080
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2400
+  # gQcomTokenSpaceGuid.PcdMipiFrameBufferAddress|0x9c400000
+
+  # Simple Init
+  gSimpleInitTokenSpaceGuid.PcdGuiDefaultDPI|350
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Samsung"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"Samsung Galaxy A71"
```
Comment (ungeskriptet) [inline Platform/Samsung/sm7150/a71.dsc:28]: Remove the vendor prefix from this line. It should be "Galaxy A71".

File: Platform/Samsung/sm7150/a71.dsc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  gRenegadePkgTokenSpaceGuid.PcdDeviceCodeName|"a71"
```
Comment (BigfootACA) [inline Platform/Samsung/sm7150/a71.dsc:29]: "a71" is not a correct codename

**Review by BigfootACA — CHANGES_REQUESTED** [review]

File: Platform/Samsung/sm7150/a71.dsc (hunk 2 above)
Comment (ungeskriptet) [reply Platform/Samsung/sm7150/a71.dsc:29]: > "a71" is not a correct codename

No, it's correct. See https://storage.googleapis.com/play_public/supported_devices.html

**Comment by ungeskriptet** [comment]
Please squash your last commit into the initial commit. Looks good to me otherwise

**Review by BigfootACA — APPROVED** [review]

File: Platform/Samsung/sm7150/a71.fdf.inc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// per-device BSP DXEs
+
+//FILE FREEFORM = 25462CDA-221F-47DF-AC1D-259CFAA4E326 {
+  //SECTION RAW = Platform/Samsung/sm7150/FdtBlob/a71.dtb
+}
```
Comment (MollySophia) [inline Platform/Samsung/sm7150/a71.fdf.inc:5]: Seems that here's a missing '//' :D

**Review by MollySophia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**